content/get-involved.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Get Involved
+---
+
+Jaeger is an open source project with [open governance](https://github.com/jaegertracing/jaeger/blob/master/GOVERNANCE.md). We welcome contributions from the community, and we’d love your help to improve and extend the project. Here are some ideas for how to get involved with the project. Some of them don't even require any coding.
+
+## No-coding involvement
+
+* Join the [online chat room](../get-in-touch) and help answering questions from the rest of the community.
+* Join our [bi-weekly video calls](../get-in-touch) to discuss issues, large initiatives, or present case studies.
+* Publish blog posts or tutorials about Jaeger, for example:
+  * What kind of deployment model you chose in your company and why.
+  * How to use Jaeger with hosted storage solutions, such as AWS Elasticsearch.
+  * What kind of problems you were able to solve with Jaeger in your organization.
+  * How did you model traces for non-trivial workflows, like async processing.
+
+  Tips:
+
+  * Tweet about your blog post at [@Jaegertracing](https://twitter.com/jaegertracing) and email to `jaeger-tracing@googlegroups.com`.
+  * If your blog is on Medium, reach out to maintainers and we may add it to our [official blog](https://medium.com/jaegertracing).
+
+* Help documenting answers to common questions, either in [Jaeger documentation](https://github.com/jaegertracing/documentation) or [Stackoverflow](https://stackoverflow.com/questions/tagged/jaeger).
+* Help improving [Jaeger documentation](https://github.com/jaegertracing/documentation), especially if you yourself run into issues where something is not clear or not working.
+* Advocate for deploying Jaeger in your company.
+* Organize local meetups to explain the benefits of Jaeger and distributed tracing.
+
+## Help with coding
+
+Of course, there's also no shortage of opportunities to help with the actual development of Jaeger. The easiest way to start is with issues tagged [good first issue](https://github.com/jaegertracing/jaeger/labels/good%20first%20issue). Note that the Jaeger project includes many different [repositories](https://github.com/jaegertracing/), covering backend components, Jaeger UI, client libraries, Kubernetes tools, analytics tools, etc. Many of them have the "good first issue" tickets, so pick whichever strikes your fancy.
```

Comment:
"strikes" -> "takes"?

Reply:
afaik the expression is "strike your fancy". I will rephrase.